### DISCUSSION TITLE
Student Developer Pack

### DISCUSSION THREAD

**Discussion by zhengqigao** [community/community#147711] [GitHub Education]
### Hi everyone,

I applied for Github education and got approved on the 19-th. I noticed that Copilot pro is free for student, and I really want to use that feature. I think it is included in the student developer pack, and I wonder when I can receive this benefit. thanks.

**Comment by WCY-dt** [comment]
I think you may get access to Copilot pro as soon as you got approved. Check [Copilot setting page](https://github.com/settings/copilot) to see if you have enabled copilot, and click on those buttons on the top to activate it in your IDEs or anything else.

**Comment by davevad93** [comment]
Hi @zhengqigao ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @zhengqigao was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by zhengqigao** [reply]
Thanks! I see. My status is still: Congrats! Your request for zhengqigao was approved and your benefits will be available soo

**Reply by davevad93** [reply]
You're welcome @zhengqigao 👍

**Comment by lpestov** [comment]
same problem, copilot is in the free plan, but student benefits are active
![image](https://github.com/user-attachments/assets/410e88af-aa81-4cee-a8db-3993f84e147d)
![image](https://github.com/user-attachments/assets/776c0852-8331-4907-b9a1-29b5538d4e66)

**Reply by davevad93** [reply]
Hey @lpestov , given the status of your application, you should have access to your benefits. Try first going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and select [Copilot](https://education.github.com/pack/redeem/copilot-student), if you can redeem it you are fine, if you can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.
